nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/model/config/builder/internal/DefaultEntityConfigBuilder.java

```diff
@@ -116,6 +120,19 @@ public DefaultEntityConfigBuilder(BeanResolverStrategy beanResolver, Map<String,
 			throw new InvalidConfigException("Persistable Entity: "+mConfig.getReferredClass()+" must be configured with @Id param which has Repo: "+mConfig.getRepo());
 		}
 		
+		if(domainSet == null)
+			return mConfig;
+		
+		/*If remote domain set is given in properties, only the alias in remote list are remote domain sets and rest are local*/	
+		List<String> remoteAliasList = domainSet.get(Scope.REMOTE.code);
+		if(CollectionUtils.isEmpty(remoteAliasList)) {
+			return mConfig;
+		}		
+		remoteAliasList.forEach(remote -> remote.trim());
+		if(remoteAliasList.contains(mConfig.getAlias())) {
+			mConfig.setRemote(true);
+		} 
+		//TODO : Add logic for checking local domainset
```

Comment:
What would happen in case of local? Could the code thrown unsupported exception till the logic for this scenario is implemented?

Reply:
I think this comment is still to be resolved

Reply:
@soham-anthem - below is the change for throwing an exception. Left the TODO comment for future implementation.